src/Analyzer.Core.JsonRuleTests/Analyzer.Core.JsonRuleTests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Add the xml comment errors to the project

Reply:
Those are only supposed to be for product code (not test code)